MediaBrowser.Model/SyncPlay/PlayQueueUpdateReason.cs

```diff
@@ -53,6 +53,11 @@ public enum PlayQueueUpdateReason
         /// <summary>
         /// A user is changing shuffle mode.
         /// </summary>
-        ShuffleMode = 9
+        ShuffleMode = 9,
+
+        /// <summary>
+        /// A user has changed the playback speed.
+        /// </summary>
+        PlaybackSpeedChange = 10
```

Comment:
Changing the playback speed shouldn't update the queue so this feels unnecessary.

Reply:
It affects Syncplay though

Reply:
From my understanding this enum is only for when the actual queue changes (item added/removed/moved etc.) not the speed of a single item. That's just the state like play/pause/volume/track etc.